### ISSUE TITLE
ConfigResolver - do we want/need it?

### ISSUE THREAD

**Issue by zboldyga** [RxNT/react-jsonschema-form-manager#18]
Is the ConfigResolver abstraction useful? The concept of a RESTConfigResolver, for instance, is just syntactic sugar over top of the fetch api, no?

Why not just have the application developer pass in a JS Promise that assembles whichever resources they want to load, in whichever way they choose to do so? This is basically what we are doing for our encounter project that uses the manager. 

```js
let configResolver = new Promise((resolve, reject) => {
   //make an API call, load a static resource, manipulate data before returning, reject if error
  resolve({ schema: 'blah' });
});

withManager(configResolver....)
//doesn't this technically do what we want?
```

**Comment by mavarazy** [comment]
Yes, I agree we can remove it.

**Comment by zboldyga** [comment]
Resolved in a few commits, version bumped to 0.3.0. Most changes happened in this commit:

https://github.com/RxNT/react-jsonschema-form-manager/commit/906a3937c29349852933695a2e1c0afa0bc2c3ec
